SonarQube.Common/ProcessRunnerArguments.cs

```diff
@@ -87,6 +130,12 @@ private static string GetQuotedArg(string arg)
             return quotedArg;
         }
 
+        private static bool ContainsSensitiveData(string text)
+        {
+            if (SensitiveDataMarkers == null) { return false; }
+            return SensitiveDataMarkers.Any(marker => text.IndexOf(marker, StringComparison.OrdinalIgnoreCase) > -1);
```

Comment:
I'm using IndexOf because I want to be able to specify the string comparison to use.